### PR TITLE
Sirdata RTD Module : bidder specific handling removal

### PR DESCRIPTION
**Pull request by nouchy**
move bidder specific handling to ortb2

## Type of change

- [X] Code style update (formatting, local variables)
- [X ] Refactoring (no functional changes, no api changes)

## Description of change
bidder specific handling removal : use ortb2 instead

### CODE REVIEW

File: modules/sirdataRtdProvider.js [hunk 1]
```diff
@@ -370,57 +369,24 @@ export function addSegmentData(reqBids, data, moduleConfig, onDone) {
           case 'msq_classic':
           case 'msq_max':
           case '366_apx':
```
Comment (dgirardi) [inline modules/sirdataRtdProvider.js:371]: I think I'm missing some context, if the intent of this is to remove bidder-specific handling why is there a giant switch statement for a bunch of bidders?

**Comment by nouchy** [comment]
Hello, the idea here is to avoid data shared with bidders directly. First step us to work through ortb2, and when bidders are ready to use SDA… please see this comment : https://github.com/prebid/Prebid.js/issues/8596#issuecomment-1548354140

**Comment by dgirardi** [comment]
> Hello, the idea here is to avoid data shared with bidders directly. First step us to work through ortb2, and when bidders are ready to use SDA… please see this comment : [#8596 (comment)](https://github.com/prebid/Prebid.js/issues/8596#issuecomment-1548354140)

I am not following; is the data different depending on the bidder? if yes, which bidders get which data should be configurable (not based in a switch statement in code); if not, you shouldn't need the switch statement. RTD modules should try to stay as agnostic as possible regarding who gets the data.

**Comment by nouchy** [comment]
We will, for sure, but we need them to be ready and move step by step, sorry. We « cleaned » what we can but still work on it…

**Comment by nouchy** [comment]
you’re right, most data IDs (segs, cats) are private, and we’re moving towards SDA and segtax ids. But the bidders don’t follow at same rate :(

**Comment by patmmccann** [comment]
@nouchy if proxistore for example isn't moving along, feel free to move your logic specific to them into their adapter.

**Comment by nouchy** [comment]
> @nouchy if proxistore for example isn't moving along, feel free to move your logic specific to them into their adapter.

Sure, I'll do it

**Review by patmmccann — APPROVED** [review]

**Pull request merged**